test/testLib.py

```diff
@@ -54,7 +54,8 @@ def testCreateUser(girder, email=None):
     assert createUser['firstName'] == firstName, 'firstName does not match'
     assert createUser['lastName'] == lastName, 'lastName does not match'
     assert createUser['login'] == login, 'login does not match, {} {}'.format(createUser['login'], login)
-    assert createUser['admin'] == False, 'user is an admin'
+    if not admin:
+        assert createUser['admin'] == False, 'user is an admin'
```

Comment:
If ```admin == True```, should we test to make sure user is in fact an admin?

If that is necessary we could get rid of the conditional and just use: 
```assert createUser['admin'] == admin```

Reply:
@henryrossiter good idea! e9d9bf7